lib/ibm_power_hmc/connection.rb

```diff
@@ -81,10 +81,20 @@ def management_console
     # @param search [Hash] The optional property name and value to match.
     # @return [Array<IbmPowerHmc::ManagedSystem>] The list of managed systems.
     def managed_systems(search = {})
-      method_url = "/rest/api/uom/ManagedSystem"
-      search.each { |key, value| method_url += "/search/(#{key}==#{value})" }
-      response = request(:get, method_url)
-      FeedParser.new(response.body).objects(:ManagedSystem)
+      if !search.empty?
+        method_url = "/rest/api/uom/ManagedSystem"
+        search.each { |key, value| method_url += "/search/(#{key}==#{value})" }
```

Comment:
This isn't up to you but from an API design it'd be nice to be able to unify these two code paths.  If the `/uom/ManagedSystem/quick` API took a `/search` and a `/All` option we could have one code path here.